src/02 - Object types.md

```diff
@@ -29,559 +32,560 @@ The following notation is used in the tables below:
+|           |        |                               |          |             |                |  |      |      |       |      |      |        |                                                                                                                                                    |
+| M001      | string | uuid                          |    x     |      x      |                |  |  cw  |  x   |   x   |      |      |        |                                                                                                                                                    |
+| M600      | date   | opprettetDato                 |    x     |      x      |                |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                                                    |
+| M601      | string | opprettetAv                   |    x     |      x      |                |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                                                    |
+| M601      | string | opprettetAvBrukerIdent        |    x     |      x      |                |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                                                    |
+| M602      | date   | avsluttetDato                 |          |             |                |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                                                    |
+| M603      | string | avsluttetAv                   |          |             |                |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                                                    |
+| M603      | string | avsluttetAvBrukerIdent        |          |             |                |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                                                    |
+| M003      | string | mappeIdent                    |    x     |      x      |                |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                                                    |
+|           | string | mappetype                     |          |             | mappetype      |  |  x   |  x   |   x   |  x   |      |  
```

Comment:
`Moetemappe`/`Abstraktmappe` and `Moeteregistrering`/`Abstraktregistrering` must be added to the list of supported links for `EksternId` and `Noekkelord`.

Reply:
Fixed